spec/spec_helper.rb

```diff
@@ -1,7 +1,13 @@
 require 'simplecov'
+require 'coveralls'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.